packages/contentstack-clone/package.json

```diff
@@ -5,22 +5,22 @@
   "author": "Contentstack",
   "bugs": "https://github.com/rohitmishra209/cli-cm-clone/issues",
   "dependencies": {
+    "@colors/colors": "^1.5.0",
     "@contentstack/cli-cm-export": "~1.11.0",
     "@contentstack/cli-cm-import": "~1.14.0",
     "@contentstack/cli-command": "~1.2.16",
```

Comment:
Please do bump the version

Reply:
@shafeeqd959 version bump already done from 1.10.0 to 1.10.1.